composer.json

```diff
@@ -44,6 +45,7 @@
         "monolog/monolog": "^1.25.3",
         "phly/phly-configfactory": "^1.0.1",
         "phly/phly-event-dispatcher": "^1.0",
+        "phly/phly-opcache-preload": "dev-master",
```

Comment:
Should this be then dev dependency?

Reply:
Honestly, I'm starting to wonder if I should make it a tool to install globally like keep-a-changelog. You'll generally use it once, when starting your project or preparing for initial deployment, and then the `preload.php` file it creates is completely standalone.

You definitely don't need it in production. Development would be fine. Or just install it once, run it, and then remove it.